File: abc160/a/a.py
def coffee(n):
    if n[2] == n[3] and n[4] == n[5]:
        return True
    else:
        return False


def main():
    s = input()

    if coffee(s):
        print("Yes")
    else:
        print("No")


if __name__ == "__main__":
    main()

File: abc160/a/a.go
package main

import "fmt"

func coffee(n string) bool {
	return n[2] == n[3] && n[4] == n[5]
}

func main() {
	var s string
	fmt.Scan(&s)

	if coffee(s) {
		fmt.Println("Yes")
	} else {
		fmt.Println("No")
	}
}
